data/en/mbs_0817.json

```diff
@@ -137,19 +137,39 @@
                             "type": "DateRange",
                             "id": "reporting-period-question-2",
                             "title": "What are the dates of the period that you will be reporting for?",
+                            "period_limits": {
+                                "minimum": {
+                                    "days": 10
+                                },
+                                "maximum": {
+                                    "days": 50
+                                }
+                            },
                             "answers": [{
                                     "type": "Date",
-                                    "id": "reporting-period-answer-from",
```

Comment:
This is piped into the following title and needs updating:

`"For the period {{format_conditional_date (answers['reporting-period-answer-from'], exercise.start_date)}} to {{format_conditional_date (answers['reporting-period-answer-to'], exercise.end_date)}}, what was the value of {{respondent.trad_as_or_ru_name}}’s <em>total turnover</em>, excluding VAT?"`